test/general/key.js

```diff
@@ -4674,11 +4675,47 @@ I8kWVkXU6vFOi+HWvv/ira7ofJu16NnoUkhclkUrk0mXubZvyl4GBg==
       expect(subkey.getAlgorithmInfo().curve).to.be.equal('nistP256');
       expect(newPrivateKey.getAlgorithmInfo().algorithm).to.be.equal('eddsaLegacy');
       expect(subkey.getAlgorithmInfo().algorithm).to.be.equal('ecdsa');
+      await subkey.verify();
```

Comment:
Is this needed? The get*Key helpers should already verify signatures internally

Reply:
This particular one is needed yeah, because the signing key we add _isn't_ selected by the call to `getSigningKey` below, and we want to make sure it's not because it's we created an invalid subkey, but rather because we prefer the other subkey's algorithm.

Some of the other ones aren't needed, but I'd prefer to keep them for simplicity and to make it clear what's broken if something fails.